trace/sync/kafka/kafka.go

```diff
@@ -0,0 +1,124 @@
+package kafka
```

Comment:
I think it makes more sense to have the old import path:

trace/kafka/sync
trace/kafka/async

and 

trace/kafka could contains common code